docs/usage.md

```diff
@@ -203,6 +205,24 @@ Skip basecalling and demultiplexing step with Guppy
 
 ## Alignment
 
+### `--protocol`
+
+Specifies the data type being inputted (DNA, cDNA or directRNA)
+
+### `--stranded`
+
+Specifies whether the inputted fastq files contain stranded data (default: false)
+
+The specific command line arguments for `graphmap` and `minimap2` using `--protocol` and `--stranded` are specified below:
+
+| `nanoseq` input              | `minimap2` presets  | `graphmap` presets |
+|------------------------------|---------------------|--------------------|
+| `--protocol DNA`             | -ax map-ont         | tba                |
+| `--protocol cDNA`            | -ax splice          | tba                |
+| `--protocol directRNA`       | -ax splice -uf -k14 | tba                |
+| `--protocol cDNA --stranded` | -ax splice -uf      | tba                |
+
+
```

Comment:
```suggestion
```

Reply:
Removing this line should get the markdownlint tests passing. Pipeline tests are purring 😃